### ISSUE TITLE
Allow for Judges Visiting from other Courts to Be Assigned to Panels/Authors

### ISSUE THREAD

**Issue by ellliottt** [freelawproject/courtlistener#630]
Our current author assignment module will not match judges to opinions when they are visiting from another court. We could update the assign_judges script to account for this, for example if it says 

"Adam and Bill, Circuit Judges; Cameron, District Judge"

Cameron will not be assigned. 

This might not be worth fixing, except manually when we notice them.

**Comment by mlissner** [comment]
Yeah, the only way I could see this working is if we know which courts are near each other geographically. I guess PostGres can do geo queries, but to do that we need the real world locations of all the court houses. Using that, though, we could expand the assign authors script to look for judges in courthouse X or courthouses within Y miles.
